backend/app/processing/server.py

```
@@ -72,16 +73,30 @@ async def process_query(user_id: str, chatbot_id: str, query: str, conversation_
+        # logger.debug(f"client history is defined {client_history}")
         # Create prompt
         prompt = create_prompt(query, context_docs, client_history)
         
-        logger.debug("Prompt is defined {prompt}")
+        # logger.debug("Prompt is defined {prompt}")
+        from processing.db_storage import DatabaseStorageManager
 
+        # 1) Grab your Vanna instance
+        vn = vanna.get_instance(chatbot_id)
+        
+        # 2) Make sure it’s pointing at the right SQLite DB
```

Ruff (0.8.2)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)